metadata-ingestion/src/datahub/entrypoints.py

```diff
@@ -11,7 +11,7 @@
 from datahub.cli.check_cli import check
 from datahub.cli.cli_utils import DATAHUB_CONFIG_PATH, write_datahub_config
 from datahub.cli.delete_cli import delete
-from datahub.cli.docker import docker
```

Comment:
Is this using a newer version of the python dependency. If yes, can you please update the `setup.py` with the minimum requirement version you tested it with?

Reply:
Could you please explain which python dependency you are asking about ?

Reply:
The version of the docker Python lib you used when testing this change.